[PATCH] Remove Error on JBoss Start

A current bug with jrs 5.5.0 causes this error after clean
install:

JasperXmlaServlet: Failed to initialize XMLA handler: java.lang.NullPointerException

This removed the problemtic servlet.
--- a/WEB-INF/web.xml
+++ b/WEB-INF/web.xml
@@ -328,18 +328,6 @@
   </servlet>
   -->
 
-  <!-- XML for Analysis Servlet -->
-  <servlet>
-    <servlet-name>JasperXmlaServlet</servlet-name>
-    <servlet-class>com.jaspersoft.jasperserver.war.xmla.XmlaServletImpl</servlet-class>
-    <load-on-startup>50</load-on-startup> 
-  </servlet>
-
-  <servlet-mapping>
-    <servlet-name>JasperXmlaServlet</servlet-name>
-    <url-pattern>/xmla</url-pattern>
-  </servlet-mapping>
-
   <!-- Status Servlet -->
   <servlet>
     <servlet-name>StatusServlet</servlet-name>
